ops-catalog/template-local.flow.yaml

```diff
@@ -2,6 +2,22 @@ import:
   - template-common.flow.yaml
 
 materializations:
+  estuary/global/inferred-schemas:
```

Comment:
Is there a reason this should be a separate materialization vs an additional binding of the exiting one? What are the considerations there?

Reply:
Had a talk with Johnny about this, the thought is that the L2 materialization should live in the `estuary/` tenant, since its purpose is to roll up all data-plane-specific L1 inferred schemas collections.

Reply:
Moved it to a binding on the existing materialization. I had it here because we were going to introduce `estuary/` tenant to hold global L2s, but also undid that per convo w/ Johnny